charts/images.yaml

```diff
@@ -103,18 +103,6 @@ images:
   repository: coredns/coredns
   tag: "1.4.0"
 
-# Alicloud Controller Manger
```

Comment:
Let's also remove `etcd-backup-restore` from this list.

Reply:
Done